planning/behavior_path_planner/src/utils/path_safety_checker/safety_check.cpp

```diff
@@ -273,8 +273,8 @@ bool checkCollision(
     const auto min_lon_length =
       calcMinimumLongitudinalLength(front_object_velocity, rear_object_velocity, rss_parameters);
 
-    const auto & lon_offset = std::max(rss_dist, min_lon_length);
-    const auto & lat_margin = rss_parameters.lateral_distance_max_threshold;
+    const auto & lon_offset = std::max(rss_dist, min_lon_length) * hysteresis_factor;
```

Comment:
@satoshi-ota 
Is it enough to consider same factor value for longtudinal/lateral direction?
I mean I don't have any specific concern about this, but if you have knowledge in this point, i just want to know!

Reply:
@kyoichi-sugahara Thanks for your comment. I think there is a possibility that we need to separate lat/lon hysteresis factor in near future. But for now, I could improve safety check chattering without that. So, I'm going to add new param if it needs separated hysteresis factor.

Reply:
@satoshi-ota 
I totally agree with your idea!
Thanks for the reply.